### PR TITLE
Bigcommerce gcp ha support

### PR DESCRIPTION
**Pull request by klizhentas**

### CODE REVIEW

File: lib/events/gcssessions/gcshandler.go [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+
+// Closer releases connection and resources associated with log if any
+func (h *Handler) Close() error {
+	h.clientCancel()
+	return h.gcsClient.Close()
+}
+
+// Upload uploads object to GCS bucket, reads the contents of the object from reader
+// and returns the target GCS bucket path in case of successful upload.
+func (h *Handler) Upload(ctx context.Context, sessionID session.ID, reader io.Reader) (string, error) {
```
Comment (russjones) [inline lib/events/gcssessions/gcshandler.go]: What happens if the session ID is the same as an existing session recording? Will it be clobbered?

**Review by russjones — APPROVED** [review]

**Comment by klizhentas** [comment]
retest this please

**Comment by klizhentas** [comment]
retest this please

**Comment by klizhentas** [comment]
retest this please

**Pull request merged**